dim/slack.py

```diff
@@ -19,11 +19,8 @@ def format_slack_notification(
     date_partition = results[0]["date_partition"]
     run_id = results[0]["run_id"]
     owner = json.loads(results[0]["owner"]).get("slack")
-    run_status = (
-        ":large_green_circle: Passed"
-        if all([result for result in results if result["passed"]])
-        else ":red_circle: Failed"
-    )
+    run_status = all([result for result in results if result["passed"]])
```

Comment:
Please keep in mind that each table is treated as separate run. This refers to all checks defined inside dim_checks.yaml for the specific dataset we're checking. So this will be marked as True if all checks for that dataset succeed (for example, consistency, uniqueness, etc).